### PR TITLE
Make deploy dependent on test

### PR DESCRIPTION
**Pull request by lauraGgit**

### CODE REVIEW

**Comment by lauraGgit** [comment]
Addresses #34

**Review by ondrae — APPROVED** [review]
The [checks](https://github.com/18F/all_sorns/pull/40/checks?check_run_id=1380585740) looks like it worked. 👍

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    branches: [main]
+    types: 
+      - completed
+
+jobs:
+  deploy:
+    name: Deploy to cloud.gov
+    needs: build_and_test
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/main'
```
Comment (ondrae) [inline .github/workflows/deploy.yml:15]: It doesn't really need this line anymore.

Comment (lauraGgit) [reply .github/workflows/deploy.yml]: 👍  removed this. @ondrae please merge if it works for you!

**Pull request merged**